File: beginner/__1000__/1015/1015.java
import java.util.Scanner;
import java.io.IOException;

public class Main{
    public static void main(String[] agrs) throws IOException {

        Scanner input = new Scanner(System.in);

        double[] data = new double[4];

        data[0] = input.nextDouble();
        data[1] = input.nextDouble();
        data[2] = input.nextDouble();
        data[3] = input.nextDouble();

        double distance = Math.sqrt(
            Math.pow((data[2] - data[0]), 2) + Math.pow((data[1] - data[3]), 2)
        );

        System.out.println(String.format("%.4f", distance));

    }
}

File: beginner/__1000__/1015/1015.lua
local x1, y1 = io.read():match("(%S+) (%S+)")
local x2, y2 = io.read():match("(%S+) (%S+)")
x1, y1, x2, y2 = tonumber(x1),tonumber(y1), tonumber(x2), tonumber(y2)

local dist = math.sqrt((x2 - x1) * (x2 - x1) + (y2 - y1) * (y2 - y1))

print(string.format("%.4f", dist))